File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/CheckoutResponse/Success.java
package com.artmurka.artmurkaapp.Model.Pojo.ItemList.CheckoutResponse;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Success {

    @SerializedName("msg")
    @Expose
    private String msg;

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public Success withMsg(String msg) {
        this.msg = msg;
        return this;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkoutresponse/Success.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkoutresponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Success {

    @SerializedName("msg")
    @Expose
    var msg: String? = null

    fun withMsg(msg: String): Success {
        this.msg = msg
        return this
    }

}